### PR TITLE
style: fixed some typos and redundant code segments

### PR DESCRIPTION
**Pull request by L2ncE**
#### What type of PR is this?

refactor

#### What this PR does / why we need it (English/Chinese):

EN：Fixed some typos and grammar errors and an extra pair of parentheses, also fixed an extra type cast
CN：修复了一些拼写错误和语法错误以及一对额外的括号，还修复了一个多余的类型转换

#### Which issue(s) this PR fixes:

Not a fix for the issue

### CODE REVIEW

File: pkg/app/context.go [hunk 1]
```diff
@@ -686,7 +686,7 @@ func (ctx *RequestContext) Set(key string, value interface{}) {
 }
 
 // Get returns the value for the given key, ie: (value, true).
-// If the value does not exists it returns (nil, false)
```
Comment (li-jin-gou) [inline pkg/app/context.go:689]: 修改注释和类型转换不算fix哈，参考一下 PR 模版改一下哈 @L2ncE

File: cmd/hz/internal/thrift/resolver.go [hunk 2]
```diff
@@ -185,7 +185,7 @@ func (resolver *Resolver) ResolveConstantValue(constant *parser.ConstValue) (mod
 	case parser.ConstType_ConstDouble:
 		return model.DoubleExpression{Src: constant.TypedValue.GetDouble()}, nil
 	case parser.ConstType_ConstLiteral:
-		return model.StringExpression{Src: string(constant.TypedValue.GetLiteral())}, nil
+		return model.StringExpression{Src: constant.TypedValue.GetLiteral()}, nil
```
Comment (li-jin-gou) [inline cmd/hz/internal/thrift/resolver.go:188]: PR 的 content 参考 https://github.com/cloudwego/hertz/pull/130 改下哈
```
What type of PR is this?
feat

What this PR does / why we need it (English/Chinese):
EN: add a warning log for invalid character in Cookie.Value
ZH：如果 Cookie.Value 中存在非法字符，则打印告警日志
```

Comment (L2ncE) [reply cmd/hz/internal/thrift/resolver.go:188]: 您再看看呢

File: pkg/app/context.go (hunk 1 above)
Comment (L2ncE) [reply pkg/app/context.go:689]: 收到

**Review by li-jin-gou — APPROVED** [review]

**Review by welkeyever — APPROVED** [review]
lgtm

**Pull request merged**